packages/desktop-client/src/components/transactions/TransactionsTable.js

```diff
@@ -2041,6 +2059,46 @@ export let TransactionTable = forwardRef((props, ref) => {
     [splitsExpanded.dispatch],
   );
 
+  let onBalanceLastSplit = useCallback(async () => {
+    const { tableNavigator, newTransactions } = latestState.current;
+    const parentTransaction = newTransactions[0];
+
+    const allSubTransactions = newTransactions.filter(
+      t => t.parent_id === parentTransaction.id,
+    );
+
+    const emptyTransactions = allSubTransactions.filter(t => t.amount === 0);
+    const remainingAmount =
+      parentTransaction.amount -
+      allSubTransactions.reduce((acc, t) => acc + t.amount, 0);
+
+    const amountPerTransaction = Math.floor(
+      remainingAmount / emptyTransactions.length,
+    );
+    let remainingCents =
+      remainingAmount - amountPerTransaction * emptyTransactions.length;
+
+    let amounts = new Array(emptyTransactions.length).fill(
+      amountPerTransaction,
+    );
+    for (let amountIndex in amounts) {
+      if (remainingCents === 0) break;
+
+      amounts[amountIndex] += 1;
+      remainingCents--;
+    }
+
+    for (const transactionIndex in emptyTransactions) {
+      await onSave({
+        ...emptyTransactions[transactionIndex],
+        amount: amounts[transactionIndex],
+      });
+    }
+
+    // cancel focused field
+    tableNavigator.onEdit(null, null);
```

Comment:
Instead of doing this, could you try passing an `error: null` on the above `onSave` call? I imaging that should prevent the `TransactionError` split component to render.